scripts/build

```diff
@@ -3,7 +3,7 @@ set -e
 
 if ! command -v dotnet-format; then
   echo "Installing dotnet formatter..."
-  dotnet tool install -g dotnet-format
+  dotnet tool install -g dotnet-format --version 4.0.0
```

Comment:
I had to lock this down since the 5.x version causes this error:
```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.1.0-rc1' was not found.
  - Check application dependencies and target a framework version installed at:
      /usr/local/share/dotnet
  - Installing .NET Core prerequisites might help resolve this problem:
      https://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
  - The .NET Core framework and SDK can be installed from:
      https://aka.ms/dotnet-download
  - The following versions are installed:
      2.2.8 at [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]

```

Reply:
Not sure -- I encountered some similar errors when I brought the dotnet integration example app into the latest Docker image. I agree with limiting the version until we can look into the broader issue.